### ISSUE TITLE
Progress bar and verbosity at Julia core runtime

### ISSUE THREAD

**Issue by SouthEndMusic** [Deltares/Ribasim#370]
As discussed at the 28-6-2023 samenwerkdag, it would be nice to receive from feedback from the Julia core during a successful run. 

- A progress bar of the simulation time would be nice. It seems to be already activated (see below), but for some reason it does not show.
```julia
    @timeit_debug to "Setup integrator" integrator = init(
        prob,
        alg;
        progress = true,
        progress_name = "Simulating",
        callback,
        config.solver.saveat,
        config.solver.dt,
        config.solver.abstol,
        config.solver.reltol,
        config.solver.maxiters,
    )
``` 

Of course we would never get an uniformly moving progress bar, because of changing timestep etc.

- It would be nice to add some verbosity options, for instance for control.

**Comment by visr** [comment]
The relevant docs are here: https://docs.sciml.ai/DiffEqDocs/stable/features/progress_bar/

Setting the global logger does the trick:

```julia
using TerminalLoggers, Logging, Ribasim
global_logger(TerminalLogger())
Ribasim.run("model.toml")
```

A quick fix would be to do this in ribasim_cli before starting the simulation: https://github.com/Deltares/Ribasim/blob/fbe786eeee331093730c56c2f3610d4d74f1b7be/build/ribasim_cli/src/ribasim_cli.jl#L29

For more advanced logging support we can look at Wflow.jl, which reads logging settings from the TOML, and can write them to both the terminal as well as a file. For interactive use you probably don't want to set the global logger, so you can just use `with_logger` like in Wflow.jl:

https://github.com/Deltares/Wflow.jl/blob/b30d32f0382e2999f4b49409b6e98e8ddfbb3656/src/Wflow.jl#L157-L158

**Comment by Hofer-Julian** [comment]
Should we close the current issue and open a new issue for this? 
> For more advanced logging support we can look at Wflow.jl, which reads logging settings from the TOML, and can write them to both the terminal as well as a file. For interactive use you probably don't want to set the global logger, so you can just use with_logger like in Wflow.jl:
> https://github.com/Deltares/Wflow.jl/blob/b30d32f0382e2999f4b49409b6e98e8ddfbb3656/src/Wflow.jl#L157-L158

**Comment by visr** [comment]
That's fine, but I don't see much benefit over keeping this issue, since it is still easy to oversee.

**Comment by evetion** [comment]
- [ ] Set verbosity level in config (to show `@info` `@debug` statements and such)
- [ ] Write to a log file as well?
